INSTALL.md

```diff
@@ -3,7 +3,7 @@
 ## Requirements
 
 **DeepEcho** has been developed and tested on
-[Python 3.7, 3.8, 3.9, 3.10 and 3.11](https://www.python.org/downloads/)
+[Python 3.8, 3.9, 3.10 and 3.11](https://www.python.org/downloads/)
```

Comment:
Just realizing that this probably shouldn't say 3.11 although it's not from your PR